tpp-app/src/home/pages/CalendarScreen.js

```diff
@@ -83,7 +91,7 @@ export default function CalendarScreen ({ navigation }) {
                     {/* This is a placeholder for the help button on final. Needed it for spacing*/}
                 </View>
             </View>
-            <Selector expanded={dropdownExpanded} views={VIEWS} selectedView={selectedView} setSelectedView={setSelectedView}/>
+            <Selector expanded={dropdownExpanded} views={VIEWS} selectedView={selectedView} setSelectedView={setSelectedView} toggleSelectedView={toggleSelectedView}/>
```

Comment:
this line here can you can replace the ```setSelectedView={setSelectedView}``` with ```toggleSelectedView={toggleSelectedView}``` instead of adding it? Because the Selector component no longer needs the setSelectedView

Reply:
oh yeah that's my b. I'll fix that rn